quodlibet/po/fr.po

```diff
@@ -1413,120 +1397,112 @@ msgid ""
 msgstr ""
 
 #: ../quodlibet/ext/events/advanced_preferences.py:68
-#, fuzzy
 msgid "Advanced Preferences"
-msgstr "Préférences du navigateur à panneaux"
+msgstr "Préférences avancées"
 
 #: ../quodlibet/ext/events/advanced_preferences.py:69
 msgid "Allow to tweak advanced config settings."
 msgstr ""
 
 #: ../quodlibet/ext/events/advanced_preferences.py:156
 msgid "I know what I'm doing"
-msgstr ""
+msgstr "Je confirme comprendre ce que je fais"
 
 #: ../quodlibet/ext/events/animosd/main.py:27
 msgid "Animated On-Screen Display"
-msgstr ""
+msgstr "Incrustation d'informations"
 
 #: ../quodlibet/ext/events/animosd/main.py:28
 msgid "Displays song information on your screen when it changes."
-msgstr ""
+msgstr "Affiche des information à l'écran dès qu'un nouveau titre est lu."
```

Comment:
"Afficher les informations concernant un titre dès qu'il est modifié ?"

What does the source English really mean?